merlin/models/tf/transforms/bias.py

```diff
@@ -206,6 +212,39 @@ def update(
     def compute_output_shape(self, input_shape):
         return input_shape
 
+    def call(
+        self, outputs: Union[Prediction, PredictionOutput], features, training=False, **kwargs
+    ):
+        if training:
+            if not isinstance(outputs, Prediction):
```

Comment:
this may be clearer if we combine the instance check to the line above (`if training and isinstance(outputs, Prediction)`)